File: ANDROID-APPS/LamseyBets/app/src/main/java/dvlp/lamseybets/MyFirebaseMessagingService.java
package dvlp.lamseybets;

import androidx.core.app.NotificationCompat;
import androidx.core.app.NotificationManagerCompat;

import com.google.firebase.messaging.FirebaseMessagingService;
import com.google.firebase.messaging.RemoteMessage;

public class MyFirebaseMessagingService extends FirebaseMessagingService {


    public void onMessageReceived(RemoteMessage remoteMessage){
        super.onMessageReceived(remoteMessage);
        showNotification(remoteMessage.getNotification().getTitle(),remoteMessage.getNotification().getBody());

    }
    public void showNotification(String title,String message){
        NotificationCompat.Builder builder=new NotificationCompat.Builder(this, "Royalbets")
        .setContentTitle(title)
                .setSmallIcon(R.drawable.logo)
                .setAutoCancel(true)
                .setContentText(message);
        NotificationManagerCompat managerCompat = NotificationManagerCompat.from(this);
        managerCompat.notify(999, builder.build());


    }
   }

File: ANDROID-APPS/LamseyBets/app/src/main/java/dvlp/lamseybets/MyFirebaseMessagingService.kt
package dvlp.lamseybets

import androidx.core.app.NotificationCompat
import androidx.core.app.NotificationManagerCompat

import com.google.firebase.messaging.FirebaseMessagingService
import com.google.firebase.messaging.RemoteMessage

class MyFirebaseMessagingService : FirebaseMessagingService() {


    override fun onMessageReceived(remoteMessage: RemoteMessage) {
        super.onMessageReceived(remoteMessage)
        showNotification(remoteMessage.notification!!.title, remoteMessage.notification!!.body)

    }

    fun showNotification(title: String?, message: String?) {
        val builder = NotificationCompat.Builder(this, "Royalbets")
                .setContentTitle(title)
                .setSmallIcon(R.drawable.logo)
                .setAutoCancel(true)
                .setContentText(message)
        val managerCompat = NotificationManagerCompat.from(this)
        managerCompat.notify(999, builder.build())


    }
}